lib/Parser/ParseFlags.h

```diff
@@ -51,6 +51,9 @@ enum
                                      //  let/const in global scope instead of eval scope so that they can be preserved across console inputs
     fscrNoAsmJs = 1 << 25, // Disable generation of asm.js code
     fscrIsModuleCode = 1 << 26, // Current code should be parsed as a module body
-    fscrAll = (1 << 27) - 1
+
+    fscrIndirect = 1 << 27,
```

Comment:
It indicates to the (eval) parser that this is an indirect eval(), in which 'new.target' should throw SyntaxError even though it is inside a function.
I will add a comment here.

---
In reply to: [73577780](https://github.com/Microsoft/ChakraCore/pull/1377#discussion_r73577780) [](ancestors = 73577780)